data/peek.json

```diff
@@ -0,0 +1,8 @@
+[
+    {
+        "speaker": "Juhani Lehtimaki",
```

Comment:
This presentation is a joint presentation with Juhani and Pierluigi. Can you please add Pierluigi as well?

Then we also need to add him to speaker's list